tests/functional/lib/helpers.js

```diff
@@ -545,6 +545,12 @@ define([
     };
   }
 
+  function openPage(context, url) {
+    return context.get('remote')
+      .get(require.toUrl(url))
+      .setFindTimeout(config.pageLoadTimeout);
```

Comment:
@shane-tomlinson I'm worried about this helper. We are not waiting for some element to finish loading.